docker-compose.yml

```diff
@@ -0,0 +1,38 @@
+version: '3.8'
+services:
+  chef-db:
+    image: mysql:8.0
+    container_name: chef-db
+    restart: always
+    environment:
+      MYSQL_USER: '${DATABASE_USERNAME}'
+      MYSQL_PASSWORD: '${DATABASE_PASSWORD}'
+      MYSQL_DATABASE: '${DATABASE_NAME}'
+      MYSQL_RANDOM_ROOT_PASSWORD: 'yes'
+    volumes:
+      - commis-data:/var/lib/mysql
+    healthcheck:
+      test: 'mysqladmin ping -h localhost'
+      interval: 5s
+      timeout: 5s
+      retries: 3
+
+  chef:
+    container_name: chef
+    env_file:
+      - local.env
+    environment:
+      - DATABASE_HOST=host.docker.internal
+    build:
+      context: ./
+      dockerfile: ./Dockerfile
+    entrypoint:
+      - /bin/sh
+    command: -c "npm run start:dev"
+    ports:
+      - '3000:3000'
+    volumes:
+      - './:/src'
+    depends_on:
+      chef-db:
+        condition: service_healthy
```

Comment:
Let's remove this one. This creates a container with the backend project but we'll not needed it by now. Just leave the `chef-db` one
```suggestion
  chef:
    container_name: chef
    env_file:
      - local.env
    environment:
      - DATABASE_HOST=host.docker.internal
    build:
      context: ./
      dockerfile: ./Dockerfile
    entrypoint:
      - /bin/sh
    command: -c "npm run start:dev"
    ports:
      - '3000:3000'
    volumes:
      - './:/src'
    depends_on:
      chef-db:
        condition: service_healthy
```